OpenLiberty/guide-reactive-messaging-sse
# Merge qa to master: Daily builds (#35)

* Update for GH Actions daily builds

* Test latest OL docker image

* Verify docker image build level

* Change for testing docker image

* Update test.yaml

Co-authored-by: Austin Seto <austin.seto@ibm.com>
Co-authored-by: Gilbert Kwan <gkwan@ca.ibm.com>
